buildozer/targets/osx.py

```diff
@@ -94,6 +94,8 @@ def build_package(self):
 
         self.logger.info('Create {}.app'.format(package_name))
         cwd = join(self.buildozer.platform_dir, 'kivy-sdk-packager-master', 'osx')
+        osx_deps = open(join(cwd, 'requirements.txt')).read()
```

Comment:
I know this is OSX and so it doesn't probably matter, but this is an anti-pattern for file reading, because the file won't be closed until the GC notices, and on some OSes this means the file is unavailable for other processes.

Reply:
Yeah, my bad. Anyway, this already fixed to `.read_text()`, witch closes descriptor.